CodeStarterCamp_Week2/main.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  main.swift
+//  CodeStarProject
+//
+//  Created by 유지완 on 2023/08/27.
+
+
+// STEP2
+import Foundation
+
+let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+var generateLotto: Set<Int> = []
+
+
+func creationLottoNumder() -> Set<Int>{
```

Comment:
```suggestion
func createLottoNumders() -> Set<Int>{
```

오해의 여지가 존재하지 않게 네이밍 해주셔야 합니다